docker/planetscale-mysql/my.cnf

```diff
@@ -0,0 +1,6 @@
+[mysqld]
+pid-file        = /var/run/mysqld/mysqld.pid
+socket          = /var/run/mysqld/mysqld.sock
+datadir         = /var/lib/mysql
+secure-file-priv= NULL
```

Comment:
Looks like this is not really needed, but that's ok, I was just curious to check
https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_secure_file_priv